test/distributed/elastic/agent/server/test/api_test.py

```diff
@@ -0,0 +1,556 @@
+#!/usr/bin/env python3
```

Comment:
Don't we need to add these files to `run_test.py` so that the tests are actually run in CI?